ui/StatusQ/src/StatusQ/Components/private/statusMessage/StatusMessageReply.qml

```diff
@@ -78,55 +80,57 @@ Item {
                             acceptedButtons: Qt.LeftButton | Qt.RightButton
                             anchors.fill: parent
                             enabled: root.profileClickable
-                            onClicked: replyProfileClicked(this, mouse)
+                            onClicked: root.replyProfileClicked(this, mouse)
                         }
                     }
                     StatusBaseText {
                         Layout.alignment: Qt.AlignVCenter
                         color: Theme.palette.baseColor1
                         font.pixelSize: Theme.secondaryTextFontSize
                         font.weight: Font.Medium
-                        text: replyDetails.amISender ? qsTr("You") : replyDetails.sender.displayName
+                        text: replyDetails.amISender ? qsTr("You") : replyDetails.sender.secondaryName || replyDetails.sender.displayName
```

Comment:
Shouldn't the display name be ok if we fix it in the model?

Reply:
Yup, correct; removed